mongo/bounties/final.json

```diff
@@ -0,0 +1,85 @@
+[
+  {
+    "_id": "60f6585b453e70eed340e8e3",
+    "season": 1,
+    "title": "Make new website landing page",
+    "description": "Description for new website landing page with specs",
+    "criteria": "A completed bounty has the following deliverables...",
+    "reward": {
+      "currency": "BANK",
+      "amount": 10000.00
+    },
+    "applicableGuilds": [
+      {
+        "guildName": "Developers Guild",
+        "id": "60f6585b453e70eed340e8e3"
+      },
+      {
+        "guildName": "Marketing Guild",
+        "id": "60f6585b453e70eed340e8e3"
+      }
+    ],
+    "createdBy": {
+      "_id": "60f6558d453e70eed340e8e0",
+      "isDaoMember": true,
+      "guildName": "Analytics Guild",
+      "discordHandle": "paul#8888",
+      "discordId": "324439902343239764"
+    },
+    "createdAt": "2021-07-20T06:40:56.112Z",
+    "dueAt": "2021-07-20T06:42:28.853Z",
+    "activatedAt": "2021-07-20T06:44:33.858Z",
+    "claimedBy": [
+      {
+        "_id": "60f6558d453e70eed340e8e1",
+        "isDaoMember": false,
+        "guildName": null,
+        "discordHandle": "rick#5555",
+        "discordId": "324423432343239764"
+      }
+    ],
+    "claimedAt": "2021-07-20T07:00:31.166Z",
+    "status": "Draft",
+    "statusHistory": [
+      {
+        "status": "Open",
+        "modifiedAt": "2021-07-20T07:00:31.166Z"
+      },
+      {
+        "status": "Draft",
+        "modifiedAt": "2021-07-20T07:00:31.166Z"
+      },
+      {
+        "status": "In-Progress",
+        "modifiedAt": "2021-07-20T07:00:31.166Z"
+      },
+      {
+        "status": "In-Review",
+        "modifiedAt": "2021-07-20T07:00:31.166Z"
+      },
+      {
+        "status": "Completed",
+        "modifiedAt": "2021-07-20T07:00:31.166Z"
+      },
+      {
+        "status": "Deleted",
+        "modifiedAt": "2021-07-20T07:00:31.166Z"
+      }
+    ],
+    "hash": "96efcf14fe28a4f17a07a5441b00285b12cbb208bf3c1bdbb030bb95d3e31a8b",
+    "skillsRequired": [
```

Comment:
I recommend we leave out skillsRequired until a future version. I think this bucket is helpful for individuals picking up a bounty to filter by, and also for analytics. However, unless the fields are carefully controlled, the utility of this data goes down as this becomes a natural language input with too many possible user inputs.

Reply:
Yep this field will be null for the MVP as it's not required. final_validation.js shows all the required fields so I think it's safe to ignore it for now